ksqldb-engine/src/main/java/io/confluent/ksql/internal/KsqlEngineMetrics.java

```diff
@@ -343,7 +343,11 @@ private void configureGaugeForState(
     final String description = String.format("Count of queries in %s state.", state.toString());
     final MetricName metricName = metrics.metricName(name, group, description, tags);
     final CountMetric countMetric = new CountMetric(metricName, gauge);
-    metrics.addMetric(metricName, gauge);
+    try {
+      metrics.addMetric(metricName, gauge);
+    } catch (IllegalArgumentException e) {
```

Comment:
This one I am not sure why we do it. It seems to be a duplicate of the other metric. They both report the state. It looks like some legacy stuff from 2018. It might be a good time to remove it?

Reply:
What other metric? cc @rodesai

Reply:
@ableegoldman these https://github.com/confluentinc/ksql/blob/81bfc1f74a4f05b0688a8f6d820808981e4c745b/ksqldb-engine/src/main/java/io/confluent/ksql/internal/KsqlEngineMetrics.java#L360